src/main/webapp/i18n/de/conversation.json

```diff
@@ -71,7 +71,7 @@
             "unArchiveChannel": {
                 "title": "Archivierung des Kanals {{ channelName }} rückgängig machen?",
                 "question": "Möchtest du die Archivierung des Kanals {{ channelName }} wirklich rückgängig machen?",
-                "description": "Mitglieder:innen können wieder Nachrichten senden. Du kannst den Kanal jederzeit wieder archivieren.",
+                "description": "Mitglieder können wieder Nachrichten senden. Du kannst den Kanal jederzeit wieder archivieren.",
```

Comment:
In #3535 you can see for example in [this diff](https://github.com/ls1intum/Artemis/pull/3535/files#diff-d7f4e57fff9e0d972bca8c68ebeba2273dffa21cb64667917eae8189fcbaaf52) that they only use "Mitglieder". An ":innen" is not necessary, because this noun has no male and female form (e.g. "Mitgliederin" does not exist). It is a neutral word "das Mitglied".